src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=RideApp
+logging.file.name=logs/app.log
+logging.file.path=logs
+logging.level.root=TRACE
```

Comment:
Nie dodajemy konfiguracji logowania. Ogólnie spoko że sobie przetestowałeś jak to działa i udało Ci się skonfigurować, ale obecnie dosyć mocno będzie nam to przeszkadzać.
Mimo wszystko dobra robota, fajnie że dałeś z siebie więcej niż mniej :)